src/components/settings/ConfigPanels/LookAndFeelConfig.tsx

```diff
@@ -475,6 +477,25 @@ export const ThemeConfigPanel = ({ bordered }: any) => {
           </StyledInputPickerContainer>
         }
       />
+
+      <ConfigOption
+        name="Default album sorting"
```

Comment:
Can you use title case here for the option name (also Sorting -> Sort)?
`Default Album Sort`

Reply:
Adjusted the title, but i think the description text should also be refined.

Reply:
What do you propose to change it to?